google/cloud/spanner/connection_options.h

```diff
@@ -52,7 +52,7 @@ class ConnectionOptions {
   /// `grpc::GoogleDefaultCredentials()`.
   ConnectionOptions& set_credentials(
       std::shared_ptr<grpc::ChannelCredentials> v) {
-    credentials_ = std::move(v);
+    if (!emulator_override_) credentials_ = std::move(v);
```

Comment:
I forgot to mention that we should probably add some documentation somewhere about this environment variable. I'm thinking that connection_options.h is probably the right spot.

Reply:
For the record, we not longer need this documentation after #1277.